web-ui/view/gitlab.ml

```diff
@@ -116,12 +122,53 @@ let list_refs ~org ~repo ~refs =
     ]
 
 let list_history ~org ~repo ~ref ~history =
-  Template.instance
-    [
-      breadcrumbs [ (prefix, prefix); (org, org) ] repo;
-      link_gitlab_refs ~org ~repo [ ref ];
-      history_v ~org ~repo ~history;
+  let head_hash =
+    match history with
+    | [] -> ""
+    | (hash, _, _, _)::_ -> hash
+  in
+  let commit_table_div =
+    div
+      ~a:
+        [ a_class [ "bg-gray-50 px-6 py-3 text-gray-500 text-xs font-medium" ] ]
+        (* TODO: We need to start with no stage separation - introduce Analysis/Checks and Build steps later *)
+      [ txt (
+        let len = List.length history in
+        if len = 1 then "Build (1)"
+        else Fmt.str "Builds (%d)" len) ]
```

Comment:
```suggestion
        let len = List.length history in
        Fmt.str "Builds (%d)" len) ]
```
If we follow the same conventions as `GitHub`, we don't have to bother about it.